base/uk.ac.stfc.isis.ibex.ui.configserver/src/uk/ac/stfc/isis/ibex/ui/configserver/editing/blocks/EditBlockDialog.java

```diff
@@ -61,6 +62,20 @@ public void propertyChange(PropertyChangeEvent evt) {
         }
     };
 
+    private PropertyChangeListener warningListener = new PropertyChangeListener() {
+        @Override
+        public void propertyChange(PropertyChangeEvent evt) {
+            for (ErrorMessageProvider model : viewModels) {
+                WarningMessage warning = model.getWarning();
+                if (warning.isWarning()) {
+                    setMessage(warning.getMessage(), 2);
```

Comment:
I think it would be clearer to write this as `setMessage(warning.getMessage(), IMessageProvider.WARNING);`.